dlp/snippets/deid/deid_test.go

```diff
@@ -263,3 +271,127 @@ func TestDeIdentifyWithWordList(t *testing.T) {
 		t.Errorf("deidentifyWithWordList(%q) = %q, want %q", input, got, want)
 	}
 }
+
+func TestDeIdentifyDeterministic(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	input := "Jack's phone number is 2255338899"
```

Comment:
question: Are we sure this is a fake phone number? Let's make sure to provide something that is obviously fake like `5555551212`.

Reply:
updated it.